### PR TITLE
Dev: fix the MySQL client package dependency

### PR DESCRIPTION
**Pull request by ccamacho**
This commit adds the correct name for the new
MySQL client.

### CODE REVIEW

**Review by jistr — APPROVED** [review]
Thanks!

File: .github/workflows/consistency-functional.yml
Comment (ccamacho) [inline .github/workflows/consistency-functional.yml:268]: This is testing the upstream review that fixes our CI let's wait for the Devstack folks to review the patch before merging this, so we can remove the workaround.

**Comment by ccamacho** [comment]
Depends on: https://review.opendev.org/c/openstack/devstack/+/823807

**Pull request merged**
